### DISCUSSION TITLE
How to convert standard elasticsearch dsl to syntax recognized by elastalert2?

### DISCUSSION THREAD

**Discussion by 1998729** [jertel/elastalert2#1006] [Q&A] [answered]
I'm now trying to create rules using platformization, but I'm not quite sure how to convert to a syntax recognized by elastalert2

**Comment by jertel** [comment, marked as answer]
See the [docs](https://elastalert2.readthedocs.io/en/latest/recipes/writing_filters.html#writing-filters-for-rules) and [examples](https://github.com/jertel/elastalert2/blob/master/examples/rules) for assistance on learning the Lucene filter syntax.
